allensdk/brain_observatory/behavior/behavior_project_cache/tables/ophys_mixin.py

```diff
@@ -11,3 +14,37 @@ def __init__(self):
             self._df = self._df.drop(
                 ['date_of_acquisition_behavior',
                  'date_of_acquisition_ophys'], axis=1)
+
+        if 'project_code_behavior' in self._df and \
+                'project_code_ophys' in self._df:
```

Comment:
It's unfortunate that we need such logic. Can you please put a comment here that the project code is stored in multiple places in the database and we need this logic to deal with that. Also it couldn't hurt to place this in a method since it's pretty long.